tests/cache/tests.py

```diff
@@ -378,6 +378,8 @@ def test_incr(self):
         self.assertEqual(cache.incr('answer', -10), 42)
         with self.assertRaises(ValueError):
             cache.incr('does_not_exist')
+        with self.assertRaises(ValueError):
+            cache.incr('does_not_exist', -1)
```

Comment:
You might as well add an assertion like this to `DummyCacheTests`'s `test_incr()` and `test_decr()` for consistency / completeness, even though it wouldn't be a regression test for your change: https://github.com/django/django/blob/5a8e8f80bb82a867eab7e4d9d099f21d0a976d22/tests/cache/tests.py#L135-L141